src/walletConnect/screens/RequestContent.tsx

```diff
@@ -0,0 +1,162 @@
+import { useNavigation } from '@react-navigation/native'
```

Comment:
this file was mostly copy + paste from the ActionRequest component - now it can be shared for session requests too